lib/voom/presenters/web_client/app.rb

```diff
@@ -25,6 +25,7 @@ class App < Sinatra::Base
         set :views, Proc.new {File.join(root, "views", ENV['VIEW_ENGINE'] || 'mdc')}
         configure do
           enable :logging
+          disable :show_exceptions if Presenters::Settings.config.presenters.root # ENV.fetch('PRESENTERS_ENABLE_ERROR_PAGE','true')!='true'
```

Comment:
I don't think this is the correct setting. Shouldn't it be `web_client.enable_error_page`?

```suggestion
          disable :show_exceptions if Presenters::Settings.config.presenters.web_client.enable_error_page
```